versions.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.26.0"
```

This looks too restrictive, I don't think the minimum required provider version needed for this module is 4.26.0. I see that the pre_authentication_banner has been added in 4.6.0.
```suggestion
      version = ">= 4.6.0"
```